docs: improve Node REPL dot commands explanation (#71)
nodejs/learn: pages/command-line/how-to-use-the-nodejs-repl.md

The REPL has some special commands, all starting with a dot `.`. They are

- `.help`: shows the dot commands help.

> - `.editor`: enters editor mode, to write multiline JavaScript code.

Reviewer: ```suggestion

- `.editor`: enters editor mode, to write multiline JavaScript code.
```

> - `.break` / `.clear`: exits multi-line code like functions. same as pressing ctrl-C.

Reviewer: ```suggestion
- `.break` / `.clear`: exits multi-line code like functions. Same as pressing CTRL-C.
```

- `.load`: loads a JavaScript file, relative to the current working directory.

> - `.save`: saves all commands you entered in the session to a file.

Reviewer: ```suggestion
- `.save`: saves all commands you entered in the session to a file.

```

> - `.exit`: exits the REPL (same as pressing ctrl-C two times).

Reviewer: ```suggestion

- `.exit`: exits the REPL (same as pressing CTRL-C twice).
```

The REPL has some special commands, all starting with a dot `.`. They are

- `.help`: shows the dot commands help.

- `.editor`: enters editor mode, to write multiline JavaScript code.
- `.break` / `.clear`: exits multi-line code like functions. Same as pressing CTRL-C.
- `.load`: loads a JavaScript file, relative to the current working directory.
- `.save`: saves all commands you entered in the session to a file.

> - 

Reviewer: ```suggestion
```

If you type `.break` at the end of a line, the multiline mode will stop and the statement will not be executed.

`.clear` clear the whole REPL context like restarting the session only in a custom REPL created programmatically:

```console
require('repl').start();
```

> 

Reviewer: Why is this added?

Author: I tested .clear in the terminal REPL, but it doesn't actually reset the REPL context, variables are stayed there. I tested it also now in a code file, the same thing so I believe I should remove this note.

Author: I have updated the code recently. Can you check it?

Author: Is there any new news?
